mubench.reviewsite/setup/TableCreationHelper.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+
+class TableCreationHelper
+{
+
+    /** @var \Illuminate\Database\Schema\Builder */
+    protected $schema;
+
+    function __construct($connection)
+    {
+        $this->schema = Schema::connection($connection);
+    }
+
+    function dropIfExistsAndCreateNewWith($modelClass, $createFunc)
```

Comment:
I'm not happy with the name of this method. I would like it to have the same name as the method below, which, in PHP, probably requires us to merge this into one method that type-checks the first parameter (Model vs. str).